source/_docs/guides/front-end-performance.md

```diff
@@ -0,0 +1,271 @@
+  </div>
+
+  <!-- 3rd pane content -->
+  <div role="tabpanel" class="tab-pane" id="tab-3-anchor2" markdown="1">
+Use an inline script in the `<body>` tag, similar to the following in order to defer scripts (replace `gaanalytics` and `//www.google-analytics.com/analytics.js` appropriately):
+
+```javascript
+<script type="text/javascript">
+  function deferJSAtOnload() {
+    var gaanalytics = document.createElement("script");
+    gaanalytics.src = "//www.google-analytics.com/analytics.js";
+    document.body.appendChild(gaanalytics);
+  }
+  if (window.addEventListener)
+    window.addEventListener("load", deferJSAtOnload, false);
+  else if (window.attachEvent)
+    window.attachEvent("onload", deferJSAtOnload);
+  else window.onload = deferJSAtOnload;
+</script>
+```
+  </div>
+</div>
+
+#### New Relic Agent
+Leave the New Relic script enabled in the `<head>` tag. Once you resolve all page rule violations for Google's test, you should see results like this:
+
+![Results](/source/docs/assets/images/speed-test-external-script.png)
+
+New Relic requires the script be placed in this tag, and it is not known to impact performance. For details on placement of this tag, see [New Relic's doc](https://docs.newrelic.com/docs/browser/new-relic-browser/page-load-timing-resources/instrumentation-browser-monitoring#javascript-placement){.external}. For details on Pantheon's free New Relic service offering, see [New Relic APM Pro](/docs/new-relic/).
+
+## Avoid Landing Page Redirects
+A redirect will add at least one extra HTTP request-response cycle. As a result, eliminating extraneous redirects can make your website more snappy. Despite your best efforts it still may be necessary to include the occasional [redirect to a primary domain](/docs/guides/launch/redirects/) using HTTPS with or without the WWW.
+
+Other considerations:
+
+- Avoid mobile-specific subdomains and use responsive web design techniques.
+- A DNS service provider such as [Cloudflare](https://support.cloudflare.com/hc/en-us/articles/200170536-How-do-I-redirect-all-visitors-to-HTTPS-SSL-){.external} may allow speedier redirects in some circumstances, but it’s still faster not to redirect at all.
+- Avoid several chained redirects that make small changes such as redirecting to HTTPS, adding or removing WWW, or adding a trailing slash. Instead, [redirect to a primary domain](/docs/guides/launch/redirects/) that has all of these standardized.
+- Pantheon doesn’t read changes to the .htaccess file or support NGINX customization, so redirections via those methods will not work.
+
+## Minify Resources (HTML, CSS and JS)
```

Comment:
@rachelwhitton @tessak22 I think `Remove Render-Blocking JavaScript and CSS` and `Minify Resources (HTML, CSS and JS)` should be next each other since they cover very similar material.

And I think `Minify` should come before `Remove Render-Blocking` because it is an easier to understand and more achievable change.

Reply:
@rachelwhitton, do you have time today to discuss this idea? I've started drafting a combined version at the bottom of https://docs.google.com/document/d/18N4hxwuopninlUHXtuTzxnLPE1qdUdfTQXpK2QQZWT0/edit?ts=59c129e9#

Reply:
absolutely! @stevector let me know when and we can sync up in slack or hop on a call. either way works for me. 

+1 on the proposed content organization changes, makes sense to group related stuff together. should help us avoid overlap as well.

Reply:
Cool, I sent an invite for 15 minutes from now.

Reply:
Sure thing! On it now